code-gurus/besty
# Chat UI

Resolves #1 
Resolves #3 

# How can be tested:
- Clone the repo and check out ft/ch-u
-  Install the dependencies: `npm install`
- Start the project by running: `expo start`
- Press `a` for IOS or `i` for Android to start the project

Using the expo client (physical device):
- Login with email and password
- Navigate to profile, and choose press on the app to open it. 

### Screenshots
<img width="430" alt="Screen Shot 2020-02-13 at 16 41 00" src="https://user-images.githubusercontent.com/20681465/74445857-b902c100-4e7f-11ea-8ecb-7b6dec7bcc94.png">

![Screenshot_1581604974](https://user-images.githubusercontent.com/20681465/74445983-e9e2f600-4e7f-11ea-9b6a-30f788f1885f.png)
